pkg/hubble/metrics/api/context.go

```diff
@@ -131,6 +135,16 @@ type ContextOptions struct {
 	// Source is the source context to include in metrics
 	Source ContextIdentifierList
 
+	// Destination is the destination context to include in metrics for egress traffic
+	DestinationEgress ContextIdentifierList
```

Comment:
Let's add a comment mentioning these are mutually exclusive with the other fields. Also, can we group Source/SourceEgress/sourceIngress and Destination/DestinationIngress/DestinationEgress together?

Reply:
done